addons/l10n_de/models/chart_template.py

```diff
@@ -16,4 +16,37 @@ def _load(self, company):
                 'external_report_layout_id': self.env.ref('l10n_din5008.external_layout_din5008').id,
                 'paperformat_id': self.env.ref('l10n_din5008.paperformat_euro_din').id
             })
+
+            outstanding_receipt = company.account_journal_payment_debit_account_id
+            outstanding_payment = company.account_journal_payment_credit_account_id
+
+            outstanding_receipt['tag_ids'] = self.env.ref('l10n_de.tag_de_asset_bs_B_II_4')
+            outstanding_payment['tag_ids'] = self.env.ref('l10n_de.tag_de_asset_bs_B_II_4')
```

Comment:
Also, should it be a `=` or a `+=`? 
Are we trying to just add the tag or to wipe out any other tag that would be in?

Reply:
Actually skr03 and 04 don't have tags for those specific lines since odoo add them at the load of the chart template, so i think a simple equals is enough ?
For the other change, you are right it would be better regarding the perf, thanks !

Reply:
I would still put a `+=` because if we ever add tag on it at somepoint, there could be a bug that we would have to hunt down here